packages/xrpl/snippets/src/bridgeTransfer.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable @typescript-eslint/consistent-type-assertions -- needed here */
+/* eslint-disable no-await-in-loop -- needed here */
+import {
+  AccountObjectsRequest,
+  LedgerEntry,
+  Client,
+  XChainAccountCreateCommit,
+  XChainBridge,
+  XChainCommit,
+  XChainCreateClaimID,
+  xrpToDrops,
+  Wallet,
+  TransactionMetadata,
+} from '../../src'
+import { CreatedNode } from '../../src/models/transactions/metadata'
+
+async function sleep(sec: number): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(resolve, sec * 1000)
+  })
+}
+
+const lockingClient = new Client(
+  'wss://sidechain-net1.devnet.rippletest.net:51233',
+)
+const issuingClient = new Client(
+  'wss://sidechain-net2.devnet.rippletest.net:51233',
+)
+const MAX_LEDGERS_WAITED = 5
+const LEDGER_CLOSE_TIME = 4
+
+void bridgeTransfer()
+
+async function bridgeTransfer(): Promise<void> {
+  await lockingClient.connect()
+  await issuingClient.connect()
+  const lockingChainDoor = 'rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4'
+
+  const accountObjectsRequest: AccountObjectsRequest = {
+    command: 'account_objects',
+    account: lockingChainDoor,
+    type: 'bridge',
+  }
+  const bridgeData = (await lockingClient.request(accountObjectsRequest)).result
+    .account_objects[0] as LedgerEntry.Bridge
+  const bridge: XChainBridge = bridgeData.XChainBridge
+  console.log(bridge)
+
+  const { wallet: wallet1 } = await lockingClient.fundWallet()
```

Comment:
I think it'd be worth adding a log here saying you're creating a wallet on the locking side as well (that way people can look it up on the explorer + reading the logs is enough to get the picture of what's happening)